.github/workflows/build.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         release: '10-2020-q4'
     - name: setup vars
       run: |
-        echo "CH_VER=stable_20.3.x" >> $GITHUB_ENV
+        echo "CH_VER=master" >> $GITHUB_ENV
```

Comment:
This is temporary. It will have to be changed to `stable_21.11.x` when the branch is available.

![image](https://user-images.githubusercontent.com/23428162/147864920-40b14daa-6f67-4420-b0c9-b86e4fd1b063.png)

Reply:
Master is a no go, we only support stable branches

Reply:
I'm adding a 21.11.x branch